### PR TITLE
Adding a new feature: Bulk Resize

### PR DESCRIPTION
**Pull request by chekoduadarsh**
I have added a new feature which can resize images in bulk with padding or the given square size, It can be highly used when we do deep learning applications to resize and clean the dataset without damaging the dataset

### CODE REVIEW

File: jarviscli/plugins/bulkresize.py [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+import glob
+
+from plugin import plugin
+
+
+def bulk_resizer(input_path, output_path, desired_size=32,
+                 color=[0, 0, 0], rename=True):
+    img_no = 0
+
+    onlyfiles = os.listdir(input_path)
```
Comment (pnhofmann) [inline jarviscli/plugins/bulkresize.py]: os.listdir definitely includes directories. Maybe filter with ``os.path.isfile``?

File: jarviscli/plugins/bulkresize.py [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    # print(onlyfiles)
+    for im_pth in onlyfiles:
+        try:
+            im = cv2.imread(im_pth)
```
Comment (pnhofmann) [inline jarviscli/plugins/bulkresize.py]: im_pth does at this point not contain the full path - but only the filename (e.g. jarvis.png).
=> This line will fail for all images not in working directory, you should prepend input_path.

Apropos, there is even in python stdlib: 
https://docs.python.org/3/library/os.path.html#os.path.join

File: jarviscli/plugins/bulkresize.py [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+               ' data collection process')
+    jarvis.say('Enter the path of directory with images to be resized :')
+    path1 = jarvis.input()
+    jarvis.say('Should I rename them to non repeating whole number series?',
+               ' y/n :')
+    rename = jarvis.input()
+    jarvis.say('Enter the path of output directory :')
+    path2 = jarvis.input()
+    jarvis.say("Enter the target size :")
+    size = jarvis.input_number()
```
Comment (pnhofmann) [inline jarviscli/plugins/bulkresize.py]: input_number default returns float - which is probably not what you want. Add parameter ``rtype=int``.

File: jarviscli/plugins/bulkresize.py [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+               'size with padding !!!! Specially designed for Deep Leanring',
+               ' data collection process')
+    jarvis.say('Enter the path of directory with images to be resized :')
+    path1 = jarvis.input()
+    jarvis.say('Should I rename them to non repeating whole number series?',
+               ' y/n :')
+    rename = jarvis.input()
+    jarvis.say('Enter the path of output directory :')
+    path2 = jarvis.input()
+    jarvis.say("Enter the target size :")
```
Comment (pnhofmann) [inline jarviscli/plugins/bulkresize.py]: If I read this correctly, does (currently) support only square images?

Comment (chekoduadarsh) [reply jarviscli/plugins/bulkresize.py]: Yes, this library is currently supporting square images!! I was mainly doing this for my deep learning project so I just made small adaptations to make it a feature in Jarvis

File: jarviscli/plugins/bulkresize.py (hunk 3 above)
Comment (chekoduadarsh) [reply jarviscli/plugins/bulkresize.py]: Done

File: jarviscli/plugins/bulkresize.py (hunk 2 above)
Comment (chekoduadarsh) [reply jarviscli/plugins/bulkresize.py]: Added

File: jarviscli/plugins/bulkresize.py (hunk 1 above)
Comment (chekoduadarsh) [reply jarviscli/plugins/bulkresize.py]: Done

**Comment by pnhofmann** [comment]
Thanks! So let's merge ;).

**Pull request merged**